backend/organization/views/organization_views.py

```diff
@@ -269,10 +274,22 @@ def post(self, request):
                     {"message": "Required parameter missing: {}".format(param)},
                     status=status.HTTP_400_BAD_REQUEST,
                 )
+        
 
+
+        if check_create_existing_name(request.data["name"]):
+            message = get_existing_name_message(request.data["name"])
+            return Response(
+                {
+                    "message": message,
```

Comment:
Here we could also send the url_slug of the existing org as an additional prop and then the frontend could add a link to the error message so users can look at the existing organization